src/scrape_up/myntra/myntra.py

```diff
@@ -0,0 +1,47 @@
+from selenium import webdriver
+from selenium.webdriver.common.by import By
```

Comment:
Do not use `selenium.webdriver`, it does not work on all systems, also on cloud platforms like EC2 and Lambda. Check for any alternatives, like `bs4`. Check if that works and let me know.

Reply:
I've researched and found that Selenium works on cloud platform. It will work on EC2 as well and since Lambda provide minimal services so it will be configured for the selenium to run but it will work.
About bs4 as I told the content of myntra loads on the runtime so selenium is used for it.